src/outputs/outputs.hpp

```diff
@@ -147,9 +149,20 @@ struct HistoryOutputVar {
       : hst_op(hst_op_), hst_fun(hst_fun_), label(label_) {}
 };
 
+struct HistoryOutputVec {
```

Comment:
I treated vector output separately from scalar history output because I didn't see another way to do this apart from a breaking change where enrolled history functions *only* return `std::vector<Real>` i.e. all existing history implementations would need to switch.